snippets/action.cson

```diff
@@ -0,0 +1,52 @@
+# Template for Action
+
+'.source.c':
+  'MOOSE action':
+    'prefix': 'action-h'
+    'body': """
+    #ifndef ${1}_H
+    #define ${1}_H
+
+    #include "Action.h"
+
+    class ${2};
+
+    template<>
+    InputParameters validParams<${2}>();
+
+    /**
+     *
+     */
+    class ${2} : public Action
+    {
+    public:
+      ${2}(const InputParameters & parameters);
+
+      virtual void act();
+    };
+
+    #endif // ${1}_H
+    """
+
+'.source.cpp':
```

Comment:
I think I changed it locally, so that language-c is loading C++ grammar instead of C one. I forgot about that ;-)